hack/dev-version/dev-version.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+
+	pluginapi "github.com/openshift/openshift-azure/pkg/api/plugin"
+)
+
+func pluginToDevVersion(pluginVersion string) string {
+	var major, minor int
+	fmt.Sscanf(pluginVersion, "v%d.%d", &major, &minor)
```

Comment:
This is wrong. This would cause e.g. `v2.2` to equal `v22`

Reply:
actually, that is exactly what we have currently. see: https://github.com/openshift/openshift-azure/tree/master/pkg/sync
v7.1 becomes the directory v71
v9.0 becomes v9
v10.0 becomes v10
no sort is going to figure that out